static/src/js/containers/MapContainer/MapWrapper.js

```diff
@@ -249,10 +205,10 @@ const MapWrapper = ({
       <Control prepend position="topright">
         {
           isFocusedCollectionPage
```

Comment:
I think its generally safer to write this like `(isFocusedCollectionPage && !isEmpty(colorMap)) && {`